databricks_sql_connector_core/src/databricks/sql/utils.py

```diff
@@ -1,5 +1,5 @@
 from __future__ import annotations
-
+import pytz
```

Comment:
This import also seems to be a leftover. If not used - please remove it

Reply:
fixed